code/game/machinery/camera/camera.dm

```diff
@@ -8,7 +8,7 @@
 	active_power_usage = 10
 	layer = FLY_LAYER
 
-	var/list/network = list("military")
+	var/list/network = list("LEADER")
```

Comment:
Why the name change? And why the caps lock?

Reply:
Because that's the network name; I didn't select it.